packages/composables/src/useCustomerOrders.ts

```diff
@@ -39,12 +58,35 @@ export function useCustomerOrders(): UseCustomerOrdersReturn {
     orders.value = fetchedOrders.data.orders.elements;
   };
 
-  const changeCurrentPage = async (pageNumber: number | string) =>
-    await loadOrders({ page: +pageNumber });
+  const loadTotalOrdersCount = async (
+    parameters: Schemas["Criteria"] = {},
+  ): Promise<void> => {
+    const fetchedOrders = await apiClient.invoke("readOrder post /order", {
+      body: parameters,
+    });
+    totalOrderItemsCount.value = fetchedOrders.data.orders.total ?? 0;
+  };
```

Comment:
can't we do the same with loadOrders method? It seems unnecessary to fetch it again only for the count